app/types/env.d.ts

```diff
@@ -1,8 +1,13 @@
 declare module 'react-native-dotenv' {
-    export const apiUrlDevelop: string;
-    export const encryptionsKey: string;
-    export const encryptionsIVKey: string;
-  }
+  export const apiUrlDevelop: string;
+  export const apiBabuniUrlDevelop: string;
+  export const encryptionsKey: string;
+  export const encryptionsIVKey: string;
+  export const nameProject: string;
+  export const projectNumber: string;
+  export const webId: string;
+  export const iosId: string;
```

Comment:
Let's make sure to add webId and iosId description into readme.md by relating that they are webClientId and iosClientId.

```
webClientId - is the OAuth 2.0 Client ID for web applications. It is required for authenticating users on Android using Firebase or Google Sign-In. It allows silent authentication and offline access.

iosClientId -  is the OAuth 2.0 Client ID specifically for iOS apps. Required for signing in with Google on iOS using Firebase or Google Sign-In. 
```